cypress/integration/onboarding/email/existing_email_adress_spec.js

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+function signupWithEmail(email, name, pw) {
+    // 2. Go to /login
+    cy.visit('/login');
+
+    // 3. Click on sign up button
+    cy.get('#signup').click();
+
+    // 4. Type 'unique.email@sample.mattermost.com' for email
+    cy.get('#email').type(email);
+
+    // 5. Type 'unique-1' for username
+    cy.get('#name').type(name);
+
+    // 6. Type 'unique1pw' for password
+    cy.get('#password').type(pw);
+
+    // 7. Click on Create Account button
+    cy.get('#createAccountButton').click();
+}
+
+describe('Email Address', () => {
+    before(() => {
+        // 1. Go to / and logout
+        cy.apiLogin('sysadmin');
+        cy.apiEnableOpenServer(true);
+        cy.apiLogout();
+    });
+
+    after(() => {
+        // Revert Enable Open Server in Team Settings config to false after test
+        cy.apiLogin('sysadmin');
+        cy.apiEnableOpenServer(false);
+    });
+
+    it('M14634 Should not create account with an existing email address', () => {
+        signupWithEmail('unique.email@sample.mattermost.com', 'unique-1', 'unique1pw');
```

Comment:
This will pass on first test but will fail next time due to email not being unique.  Can you make the email and username unique so that it will still pass even on multiple tests run?

Reply:
This had crossed my mind and I did look around to delete the user after test run but couldn't find a way to do that with API/UI functions, since DELETE is devised to only "Deactivate" a user on /api/v4/users/{userID}. I can definitely add a unique email to make this run on multiple test runs :)